### PR TITLE
Fixed chat message subject.

### PR DESCRIPTION
**Pull request by AndreyTepaykin**

### CODE REVIEW

File: platform/plugins/Streams/web/js/Streams.js [hunk 1]
```diff
@@ -4835,26 +4835,28 @@ Q.onInit.add(function _Streams_onInit() {
...
 			var content = Q.getObject(["content"], message);
-
-			// if this message type absent in config
-			if (!Q.getObject([messageType], notificationsAsNotice)) {
-				return;
+			var messageUrl = message.getInstruction('invitedUrl') || message.getInstruction('url');
+			var noticeOptions = Q.getObject([messageType], notificationsAsNotice);
+			var pluginName = messageType.slice(0, messageType.indexOf('/'));
+
+			// Travel plugin settings
```
Comment (EGreg) [inline platform/plugins/Streams/web/js/Streams.js]: No! Streams plugin shouldn't know about Travel. Or passengerId. If you want you can make some sort of hook here (Q.Event can be triggered, and if it returns false, then you do `if (false === Streams.onPostEvent.handle(pluginName)) { return }` ... 

but seriously, what are you doing here?
why are you skipping messages from other passengers?

File: platform/plugins/Streams/web/js/Streams.js [hunk 2]
```diff
@@ -4835,26 +4835,28 @@ Q.onInit.add(function _Streams_onInit() {
...
 			var messageType = Q.getObject(["type"], message);
 			var byUserId = Q.getObject(["byUserId"], message);
 			var content = Q.getObject(["content"], message);
-
-			// if this message type absent in config
-			if (!Q.getObject([messageType], notificationsAsNotice)) {
-				return;
+			var messageUrl = message.getInstruction('invitedUrl') || message.getInstruction('url');
+			var noticeOptions = Q.getObject([messageType], notificationsAsNotice);
+			var pluginName = messageType.slice(0, messageType.indexOf('/'));
```
Comment (EGreg) [inline platform/plugins/Streams/web/js/Streams.js]: or you can do `messageType.split('/')[0]`

**Review by EGreg — CHANGES_REQUESTED** [review]

File: platform/plugins/Streams/web/js/Streams.js (hunk 1 above)
Comment (AndreyTepaykin) [reply platform/plugins/Streams/web/js/Streams.js]: I removed "Travel plugin settings" while. Will do it later.
I need some messages for particular passengers, for example "Driver arrive to your location". This message meant only for one passenger.

File: platform/plugins/Streams/web/js/Streams.js (hunk 2 above)
Comment (AndreyTepaykin) [reply platform/plugins/Streams/web/js/Streams.js]: Yes you right. Fixed in last commit.

**Review by EGreg — APPROVED** [review]

**Pull request merged**